src/entities/Proposal/types.ts

```diff
@@ -475,6 +477,74 @@ export const newProposalPitchScheme = {
   },
 }
 
+export type NewProposalTender = {
+  linked_proposal_id: string
+  project_name: string
+  summary: string
+  problem_statement: string
+  technical_specification: string
+  use_cases: string
+  deliverables: string
+  target_release_date: Date
+  coAuthors?: string[]
+}
+
+export const newProposalTenderScheme = {
+  type: 'object',
+  additionalProperties: false,
+  required: [
+    'linked_proposal_id',
+    'project_name',
+    'summary',
+    'problem_statement',
+    'technical_specification',
+    'use_cases',
+    'deliverables',
```

Comment:
We might be missing target release date here, I'm not sure if it's required 👁️

Reply:
You are right!